### ISSUE TITLE
webpack "Can't resolve 'can-route-mock'"

### ISSUE THREAD

**Issue by cherifGsoul** [canjs/webpack-example#14]
* cloning the repo
* checkout ```can-5``` branch
* change ```can``` dep to stable
* run ```npm run  develop``` or run ```npm run build``` the console output:
```ERROR in ./node_modules/can/es/can-route-mock.js
Module not found: Error: Can't resolve 'can-route-mock' in '/home/cherif/server/donejs/webpack-example/node_modules/can/es'
 @ ./node_modules/can/es/can-route-mock.js 1:0-39 1:0-39
 @ ./node_modules/can/ecosystem.js
 @ ./node_modules/can/can.js
 @ ./index.js
```

**Comment by matthewp** [comment]
Caused by https://github.com/canjs/can-route-mock/issues/1

**Comment by cherifGsoul** [comment]
Fixed in `can@5.1`!
